ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -432,6 +442,24 @@ impl Mutation {
             "Only admin is able to update oauth credential",
         ))
     }
+
+    async fn update_email_setting(
+        ctx: &Context,
+        smtp_username: String,
+        smtp_password: Option<String>,
+        smtp_server: String,
+    ) -> Result<bool> {
+        ctx.locator
+            .email_setting()
+            .update_email_setting(smtp_username, smtp_password, smtp_server)
+            .await?;
+        Ok(true)
+    }
+
+    async fn delete_email_setting(ctx: &Context) -> Result<bool> {
```

Comment:
```suggestion
    async fn delete_email_setting(ctx: &Context) -> Result<()> {
```

Reply:
This is not allowed, `()` does not implement a trait required in order to return it from a graphql endpoint. A `bool` which is always true is used everywhere else for this.